LayoutTests/imported/w3c/web-platform-tests/css/selectors/user-invalid.html

```diff
@@ -12,6 +32,31 @@
   assert_false(input.matches(':user-invalid'));
 }, ':user-invalid selector should be supported');
 
+promise_test(async () => {
+  const input = document.querySelector("#user-invalid-test");
+  assert_false(input.matches(":user-valid"), "Initially does not match :user-valid");
+  assert_false(input.matches(":user-invalid"), "Initially does not match :user-invalid");
```

Comment:
The previous subtest could probably be combined with this one (moving the input validity check) and the other `<input>` removed.